main.js

```diff
@@ -0,0 +1,55 @@
+/*task 6*/
+let user = 'John Doe';
+let student = 'Irina';
+
+console.log(user);
+console.log(student);
+
+user = student /*Irina*/
+
+console.log(user);
+
+/*task 7*/
+let test = 1;
+
+test++ /*2*/
+test = test + '1' /*21*/
+
+console.log(test);
+
+test = test - 1 /*0*/
+
+console.log(test);
+
+test = true; /*true*/
+
+console.log(test);
+
+/*task 8*/
+let arr = [2, 3, 5, 8];
+let result = 1;
+let i = 0;
+
+for (; i < arr.length; i++) {
```

Comment:
Why not to move `let i = 0` into the place of the missed argument here?

Reply:
I will do so. made different choices.